src/combinatorics/configuration.lean

```diff
@@ -209,4 +211,22 @@ lemma has_points.card_le [has_points P L] [fintype P] [fintype L] :
   fintype.card L ≤ fintype.card P :=
 @has_lines.card_le (dual L) (dual P) _ _ _ _
 
+variables {P L}
```

Comment:
These are still implicit from your change above, but I guess they should be explicit in `has_points.card_le`.

Reply:
@robertylewis Prior to this line, P and L are explicit, due to the variables line above `has_lines.card_le` (~line 180).